airflow/contrib/hooks/telegram_hook.py

```diff
@@ -0,0 +1,88 @@
+import telegram
+from time import sleep
+from airflow.hooks.base_hook import BaseHook
+from airflow.exceptions import AirflowException
+
+
+def telegram_retry(func, *args, **kwargs):
+    max_retries = kwargs.pop("max_retries", 5)
+    retry_sleep = kwargs.pop("retry_sleep", 0)
+    # perform request with retry
+    for retry in range(max_retries):
+        try:
+            return func(*args, **kwargs)
+        except (telegram.error.TimedOut,) as e:
```

Comment:
Why only this error? What if an internal server error happens? Then you also need to retry.